uwblueprint/sistering
# Implement admin "select shift times" calendar

## Ticket link

Closes #102 



## Implementation description
* Leveraged FullCalendar to create the admin shifts calendar.
* To use the calendar:
- * Click to create a new event
- * Drag the bottom of the event card to change the length of the event
- * Click on the card to prompt deletion modal



## Steps to test
1. Navigate to `http://localhost:3000/admin/posting/create/shifts` and play around with the calendar!
2. Feel free to add a `console.log(events)` in the code to check the output data



## What should reviewers focus on?
* Correct usage of FullCalendar
* Intuitive feel to the calendar


## Checklist
